### ISSUE TITLE
Change "Unpublished" text if content is scheduled?

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#3077]
## Describe your issue or idea
Should the default "This content is unpublished" text be adapted if the content is unpublished but scheduled for publication? (e.g. "This Post is scheduled to be published.")

![screenshot from 2018-05-02 21-48-54](https://user-images.githubusercontent.com/871421/39558416-1421f0a2-4e54-11e8-8e38-d7e92fb955cd.png)

Edit: moved the second part to https://github.com/backdrop/backdrop-issues/issues/3145

----

PR: https://github.com/backdrop/backdrop/pull/2199

**Comment by serundeputy** [comment]
PR: https://github.com/backdrop/backdrop/pull/2199

<img width="1634" alt="screen shot 2018-05-19 at 1 36 26 pm" src="https://user-images.githubusercontent.com/6763671/40271323-bd9646b6-5b69-11e8-90b4-7627ce535aaf.png">

**Comment by olafgrabienski** [comment]
Re viewing the content, should we also change the background-color, for example to yellow or orange? Here is an orange example (`background-color: #ff6d00;`):

![screen-backdrop-pr-2199-scheduled-orange](https://user-images.githubusercontent.com/14188307/40279891-fe6010d2-5c4a-11e8-9c7a-bd58cc588d8f.png)

Actually, I like the orange but I guess we should consider to use a color scheme for such cases (like the messages). Thoughts?

**Comment by quicksketch** [comment]
> This works great for the first location (notification when viewing the content), but does not address the status column in the main admin content page. Should that be a separate issue?

Yes, probably. That is not trivial to set up in Views currently because the status of the node is either 0 or 1. A scheduled piece of content has a status of 0 but a `scheduled` property that is a timestamp, so we'd need to combine these two fields to determine if content is unpublished, published, or scheduled.

**Comment by laryn** [comment]
@olafgrabienski I like the idea of changing the color but am not sure about the orange.

@quicksketch I moved the second part over here: https://github.com/backdrop/backdrop-issues/issues/3145

**Comment by quicksketch** [comment]
Posted feedback: https://github.com/backdrop/backdrop/pull/2199#pullrequestreview-123515177

I'd prefer to hold off on the color change. IMO the indication is the same: that content is not available to end-users. Though the color indication of scheduled vs. unpublished may be handy for experienced editors skimming the page, saving them from reading the message.

**Comment by olafgrabienski** [comment]
I agree not to change the color.

**Comment by serundeputy** [comment]
In the current PR I've moved the functionality out of the `basis` theme and into `node.theme.inc`
@quicksketch feels it should remain in `basis`.
I said it should be available to anyone regardless of theme.  Anyone else have opinion?
https://github.com/backdrop/backdrop/pull/2199#discussion_r191011322

- [ ] Decide on moving the functionality to `node.theme.inc` or leave in `basis`
- [ ] We need to change the date to use the `medium` date format.

**Comment by laryn** [comment]
It does seem to me to be more general than the basis theme and I would support it in `node.theme.inc`.

**Comment by laryn** [comment]
I think it is useful for any theme but would rather have it in for Basis than not in at all.

**Comment by olafgrabienski** [comment]
I think also the message is useful for any theme.

@laryn I've had a look at the issue which introduced the visual indication of "unpublished" for *Basis*. Found a few statements re limiting it to *Basis* there:

(1) https://github.com/backdrop/backdrop-issues/issues/2269#issuecomment-299775237:

> I kept the code contained within Basis as it's fairly minimal and highly opinionated (e.g. using the word "Draft" instead of "Unpublished", z-indexing, styling, etc.)

(2) https://github.com/backdrop/backdrop-issues/issues/2269#issuecomment-316809670:

> However, if we want to add the content type name into the 'Unpublished X' text (...)
We could add something in via preprocess for unpublished nodes, but that would be a Basis-only markup change, where all the other themes depend on only the existing CSS class.

I guess, the first statement isn't necessarily valid anymore, not sure about the second.

**Comment by BWPanda** [comment]
I'm surprised that the `.unpublished-indicator` element isn't available to all themes. I think the markup should be available to all, and then individual themes can choose to style (or hide) it however they see fit. But maybe that should be a seperate issue and this one just focus on the wording...

As for the wording, I think that it should still be stated that the content is unpublished. So something like:

> This [type] is unpublished, but is scheduled for publication on [date].
